components/camel-cm/src/main/java/org/apache/camel/component/cm/exceptions/MessagingException.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.cm.exceptions;
+
+public class MessagingException extends CMDirectException {
```

Comment:
Claus,

Good points. I will change it asap.
Thanks for the feedback.

Omar
El 21 mar. 2016 9:23, "Claus Ibsen" <notifications@github.com> escribió:

> In
> components/camel-cm/src/main/java/org/apache/camel/component/cm/exceptions/MessagingException.java
> <https://github.com/apache/camel/pull/911#discussion_r56790775>:
>
> > + * this work for additional information regarding copyright ownership.
> > + * The ASF licenses this file to You under the Apache License, Version 2.0
> > + * (the "License"); you may not use this file except in compliance with
> > + * the License.  You may obtain a copy of the License at
> > + *
> > + *      http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +package org.apache.camel.component.cm.exceptions;
> > +
> > +public class MessagingException extends CMDirectException {
>
> Do not create your own exceptions for common errors such as invalid / url
> etc. Use the exceptions from camel itself.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/camel/pull/911/files/6fb1026053a138b896c7a49b64ebe19558eb8a20#r56790775>
>